src/main/java/com/zaxxer/hikari/pool/PoolEntry.java

```diff
@@ -98,7 +112,21 @@ public void setKeepalive(ScheduledFuture<?> keepalive) {
 
    Connection createProxyConnection(final ProxyLeakTask leakTask)
    {
-      return ProxyFactory.getProxyConnection(this, connection, openStatements, leakTask, isReadOnly, isAutoCommit);
+      Connection newproxyconn= ProxyFactory.getProxyConnection(this, connection, openStatements, leakTask, isReadOnly, isAutoCommit);
+      // beginRequest Call just to let driver know that the connection has been borrowed from the pool.
```

Comment:
Do away with these comments as they aren't consistent with the existing codebase.

Reply:
I have remove the comments. Thank's for letting me know